src/main/java/com/gooddata/md/MetadataService.java

```diff
@@ -97,6 +101,31 @@ public MetadataService(RestTemplate restTemplate) {
     }
 
     /**
+     * Retrieves a collection of objects corresponding to the supplied collection of URIs.
+     *
+     * @param project project that contains the objects to be retrieved
+     * @param uris collection of URIs
+     * @return collection of metadata objects corresponding to the supplied URIs
+     */
+    public Collection<Obj> getObjsByUris(Project project, Collection<String> uris) {
+        notNull(project, "project");
+        notNull(project.getId(), "project.id");
+        notNull(uris, "uris");
+
+        try {
+            final BulkGet result = restTemplate.postForObject(BulkGet.URI, new BulkGetUris(uris), BulkGet.class, project.getId());
+
+            if (result != null) {
+                return result.getItems();
+            } else {
+                throw new GoodDataException("empty response from API call");
```

Comment:
Let's make it consistent with other error messages: capital first letter, sentence structure

Reply:
heh :) keeping it consistent was what I tried - just look for "empty response from API call" in that file! :)

Reply:
Yeah I looked into a different file... Apparently there is no consensus on this matter. Nevermind then.